### PR TITLE
Added three latest tocall IDs and updated Aprx Info

### PR DESCRIPTION
**Pull request by PhirePhly**
I couldn't figure out how to distill the derivative formats from the yaml file, so those need to be updated before this is finished.

### CODE REVIEW

File: tocalls.yaml [hunk 1]
```diff
@@ -466,6 +466,10 @@ tocalls:
    model: APRS/CE
    class: app
 
+ - tocall: APCDS0
+   vendor: Leon Lessing, ZS6LMG
+   model: Cell Tracker
+
```
Comment (hessu) [inline tocalls.yaml]: APCDS0 is not in tocalls.txt actual index, and seems like an undeployed client. Let's wait for him to release, or let him add it himself.

File: tocalls.yaml [hunk 2]
```diff
@@ -495,6 +495,11 @@ tocalls:
    os: ios
    class: app
 
+ - tocall: APFPRS
+   vendor: Jeroen Vreeken, PE1RXQ
+   model: FreeDV
+   class: software
```
Comment (hessu) [inline tocalls.yaml]: Let's leave it to Jeroen to provide the info, then, if he wishes to release the client and provide contact info. Thanks!

**Comment by tonyc** [comment]
@PhirePhly I came here looking for the updates for the TH-D74 - any chance you can update this PR with @hessu's suggested changes to get it merged? Thanks!

**Comment by PhirePhly** [comment]
I was assuming he'd cherrypick the two patches which didn't have issues, but I can create a new PR when I get home. I still feel bad that I can't offer a PR which includes all the other formats updated, but my perl-fu is weak and I can't figure out how to get the validate script running (which I assume also distills the other formats).

**Comment by hessu** [comment]
I was assuming I'd cherrypick the two patches too, but just didn't get around to doing it yet. And, I like to merge a PR with just the yaml changes and run the script myself; it's easier to review that way.

**Pull request closed without merging**